database/migrations/2019_06_10_220851_add_work_env_and_culture_to_job_poster.php

```diff
@@ -0,0 +1,110 @@
+<?php
+
+use Illuminate\Support\Facades\Schema;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+class AddWorkEnvAndCultureToJobPoster extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::table('job_posters', function (Blueprint $table) {
+            $table->integer('team_size')->nullable();
+
+            // Work Env checkboxes
+            // Storing as a json because these features are likely to change frequently
+            // This way the frontend can store and check whatever tags it wants to.
+            $table->json('work_env_features')->nullable();
```

Comment:
Should add the proposed shape of the json as a comment in the `JobPoster` model and delete the comments here.

Reply:
Addressed in d5f3d45ee726b00494c204a3f1dc72bcbb0e5e80